pipelines/vars/forklift_katello.yml

```diff
@@ -4,7 +4,7 @@ server_box:
   ansible:
     variables:
       foreman_server_repositories_katello: true
-      foreman_server_repositories_ostree: true
+      foreman_server_repositories_ostree: false
```

Comment:
I am worried this will break older pipelines

Reply:
We don't run any ostree *tests*, so probably not. But also enabling those repos on current nightly has no harm, so I'd leave that `true` for now?